client/src/components/ReviewTile.js

```diff
@@ -1,13 +1,31 @@
 import React from "react";
 
-const ReviewTile = ({ review }) => {
-  
+const ReviewTile = ({ user, review, deleteReview }) => {
+  const handleReviewDelete = () => {
+    if (confirm("Are you sure you want to delete this review? \nThis cannot be undone") == true) {
+      deleteReview(review)
+    } else {
+      alert("This review has NOT been deleted.")
+    }
+  }
+
+  let deleteButton = ''
+  let editButton = ''
+  if (user?.is_admin == true) {
```

Comment:
javascript prefers camelCase over snake_case - `is_admin` should instead be `isAdmin`